app/javascript/components/pages/admin/form-builder/components/field-dialog/component.spec.js

```diff
@@ -125,16 +109,10 @@ describe("<FieldDialog />", () => {
     });
 
     it("should render the FieldDialog with OrderableOptionsField", () => {
-      const { component } = setupMockFormComponent(FieldDialog, {
-        props: { mode: "edit", formId: "field-dialog-form" },
-        state: initialStateSelectField
-      });
-
-      expect(component.find(FieldDialog)).to.have.lengthOf(1);
-      expect(component.find(OrderableOptionsField)).to.have.lengthOf(1);
-      expect(component.find(DraggableOption)).to.have.lengthOf(3);
-      expect(component.find(FieldDialog).find(DialogTitle).text()).to.equal(`fields.edit_label`);
-      expect(component.find(DraggableOption).find(SwitchInput)).to.have.lengthOf(3);
+      const props = { mode: "edit", formId: "field-dialog-form" };
+
+      mountedComponent(<FieldDialog {...props} />, initialStateSelectField);
+      expect(screen.getByText("fields.edit_label")).toBeInTheDocument();
```

Comment:
Since `fields.edit_label` is part of the `FielDialog`, we might need a `data-testid` to make sure the `OrderableOptionsField` component is rendered.